### DISCUSSION TITLE
Documentation for the options in the Image comparison library.

### DISCUSSION THREAD

**Discussion by liorc955** [Visual-Regression-Tracker/Visual-Regression-Tracker#371] [Q&A] [answered]
Hey,
There is any documentation regarding the options in the Image comparison library list (especially for the odiff option?)
![image](https://user-images.githubusercontent.com/37208468/183944231-e58322e3-d0ac-43f0-ac80-eeeb4a3ba27b.png)

**Comment by pashidlos** [comment, marked as answer]
@liorc955 there is not any documentation 
could you share what you have in mind as example of cases to be documented?

presented libs are OSS and you could find the corresponding info regarding it
odiff is not working atm due to https://github.com/dmtrKovalenko/odiff/issues/55

**Reply by liorc955** [reply]
I'm new at visual testing so I guess these are common methods that I should go deep and learn.
Thank you for your response.

**Reply by pashidlos** [reply]
the idea of service is that you don't need to do that 
everything is wrapped and prepared to use 

regarding the deep understanding of image comparison algorithms you could refer to respective image comparison lib
